mathlibtools/leanproject.py

```diff
@@ -205,6 +205,22 @@ def get_mathlib_cache(force: bool = False) -> None:
     except (LeanDownloadError, FileNotFoundError) as err:
         handle_exception(err, 'Failed to fetch mathlib oleans')
 
+@cli.command()
+def delete_zombies() -> None:
+    """Delete zombie oleans, .olean files with no matching .lean files"""
+    try:
+        proj().delete_zombies()
+    except (LeanDownloadError, FileNotFoundError) as err:
+        handle_exception(err, 'Failed to delete zombies')
+
+@cli.command()
+def clean() -> None:
+    """Delete all olean files"""
+    try:
+        proj().clean()
+    except (LeanDownloadError, FileNotFoundError) as err:
```

Comment:
Why do you want a specific error message for those two exceptions, and how LeanDownloadError could arise here?